src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/jpa/repository/InterventionFilterRepositoryImplTest.kt

```diff
@@ -148,11 +148,50 @@ class InterventionFilterRepositoryImplTest @Autowired constructor(
     interventionFactory.create(contract = dynamicFrameworkContractFactory.create(npsRegion = npsRegionFactory.create()))
     interventionFactory.create(contract = dynamicFrameworkContractFactory.create(pccRegion = pccRegionFactory.create(id = "testID", name = "testName")))
     interventionFactory.create(contract = dynamicFrameworkContractFactory.create(allowsMale = false, pccRegion = pccRegionFactory.create()))
-    val found = interventionFilterRepositoryImpl.findByCriteria(listOf("avon-and-somerset"), allowsFemale = true, allowsMale = false)
+    val found = interventionFilterRepositoryImpl.findByCriteria(listOf("avon-and-somerset"), allowsFemale = true, allowsMale = false, null, null)
 
     assertThat(found.size).isEqualTo(1)
     assertThat(found[0].dynamicFrameworkContract.allowsFemale).isTrue
     assertThat(found[0].dynamicFrameworkContract.allowsMale).isFalse
     assertThat(found[0].dynamicFrameworkContract.pccRegion?.id).isEqualTo("avon-and-somerset")
   }
+
+  @Test
+  fun `get interventions filtering by minimum age on exact match`() {
```

Comment:
i'm slightly surprised by exact match. perhaps this has already been discussed and i missed it, but i would expect minimum age to apply a greater than expression i.e. `select ... from intervention i inner join dynamic_framework_contract d on i.dynamic_framework_contract_id = d.id where d.minimum_age >= 28;`

Reply:
Ryan Forsyth  9:44 AM
I know we've gone over this a bunch of times but on filtering by age; say minimumAge = 18 was passed through. Are we looking for exact matches on minimum age = 18?

Lawrence Forooghian  9:44 AM
That's how I'm interpreting it at the moment
That it's all equality, no way to specify an inequality

Ryan Forsyth  9:45 AM
okay sounds good 
Thanks

Rhiannon James  10:21 AM
Exact match at the time of referral is my understanding @Ryan Forsyth